.gitignore

```diff
@@ -26,6 +26,7 @@ yarn-error.log*
 # env files
 .env
 .env.local
+.env.example
```

Comment:
This should not be in gitignore
This is for the developer to know what data should be in the .env file.

Reply:
Hi @tanvirstreame ,

The file will be visible, but new users will be prevented from accidentally pushing their keys.

Reply:
Hey @Yaronglp This `env.example` is for giving ideas about what should be in `env`. Also, keeping the value in `env.example` will not affect anything in the application.

Reply:
agree with @tanvirstreame